java/mdsplus-api/src/main/java/mds/data/DTYPE.java

```diff
@@ -30,7 +30,7 @@ public enum DTYPE
 	OU("Octaword_Unsigned", "OU"), // 25
 	O("Octaword", "O"), // 26
 	G("G_Float", "G"), // 27
-	H("H_Float", "H"), // 28
+	H("H_Float", "H"), // 28 /*bad*/
```

Comment:
Does this comment mean that H_Float still doesn't work?

Reply:
there does not seem to be support for h_float anymore. so jtraverser2 cannot use it either. if i remember correctly i tried to craft an H_FLOAT value in tdi and failed.

Reply:
Works for me, just wanted to call it out explicitly, I saw the error handling for it too which looks good :+1: